services/oauth/v2/http/transport.go

```diff
@@ -106,8 +106,11 @@ func (t *OAuthTransport) preRoundTrip(rts *roundTripState) *http.Response {
 		rts.req = rts.req.WithContext(cntx.CtxWithSecret(rts.req.Context(), authResponse.Account.Secret))
 		err = t.Augmenter.Augment(rts.req, body, authResponse.Account.Secret)
 		if err != nil {
-			t.log.Debugn("augmenting the secret",
-				logger.NewErrorField(err))
+			t.log.Errorn(fmt.Sprintf("augmenting the secret pre roundTrip: %s", err.Error()),
```

Comment:
you can place error on field instead of formatting it. It makes it easier to filter, aggregate and place alerts on status message while the error might differ slightly

Reply:
Got it. Makes sense. Will make relevant changes

Reply:
This is done